python/cudf/cudf/_lib/aggregation.pyx

```diff
@@ -30,6 +30,7 @@ from cudf._lib.types import Interpolation
 
 cimport cudf._lib.cpp.aggregation as libcudf_aggregation
 cimport cudf._lib.cpp.types as libcudf_types
+from cudf._lib.cpp.aggregation cimport underlying_type_t_correlation_type
```

Comment:
This typedef should probably go in `cudf._lib.types` where all the other `underlying_*` typedefs live.

Reply:
Reason for adding this type to `aggregation.pxd`, is that it is not used at any other place besides aggregations.
didn't want to overload `types.pxd`, which gets included in most files (and hence to avoid extra compilations).